index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Campaign Gaming Summit</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="asset/mouse.png">
+</head>
+<body>
+    <div id="overlay"></div>
+    <div id="mobile-menu" class="mobile-main-menu flex-col">
+        <img src="asset/fire.png" alt="">
+        <ul class="flex-col">
+            <li class="flex-row"><a href="index.html" class="menu-item">Home</a></li>
+            <li class="flex-row"><a href="about.html" id="link1" class="menu-item">About</a></li>
+            <li class="flex-row"><a href="#program" class="menu-item">Program</a></li>
+            <li class="flex-row"><a href="#speaker-section" class="menu-item">Speakers</a></li>
+            <li class="flex-row"><a href="about.html" class="menu-item">Sponsors</a></li>
+            <li class="flex-row"><a href="#" class="menu-item">News</a></li>
+        </ul>
+    </div>
+    <header>
+        <div id="top-bar">
+            <ul>
+                <li>
+                    <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                </li>
+                <li>
+                    <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                </li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <div id="navbar">
```

Comment:
- [ ] Instead of taking a `nav` element inside this div, kindy consider taking nav element here instead of `div` to add semantic meaning to your navbar.
- [ ] I hope you will do the same in the about page.